### PR TITLE
Request for a Ubuntu instance c4.8xlarge

### PR DESCRIPTION
**Pull request by itismeghasyam**
- To extract elevateMS features
- Used the latest Ubuntu AMI (US West Oregon) from http://www.louisaslett.com/RStudio_AMI/

### CODE REVIEW

File: config/prod/elevatems_features.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v8.j2
```
Comment (zaro0508) [inline config/prod/elevatems_features.yaml]: this is for aws linux distro.  you'll want to use the ubuntu template: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/147/files

File: config/prod/elevatems_features.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "c4.8xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "240"
+  # (Optional) Base instance on AMI (default: ami-082278746f893d99c) (supported distros: AWS linux)
+  AMIId: "ami-01a01f46103fb6737"
```
Comment (zaro0508) [inline config/prod/elevatems_features.yaml]: is this a new AMI from your personal account? has it been shared to the scicomp account? is the root volume encrypted?

File: config/prod/elevatems_features.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+
+#sceptre_user_data:
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+
+# For CI system (do not change)
+hooks:
+  before_create:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/managed-ec2-v8.j2 --create-dirs -o templates/remote/managed-ec2-v8.j2"
```
Comment (zaro0508) [inline config/prod/elevatems_features.yaml]: make sure this and below are referencing ubuntu template as well.

File: config/prod/elevatems_features.yaml (hunk 1 above)
Comment (itismeghasyam) [reply config/prod/elevatems_features.yaml]: I changed the template to the Ubuntu one

File: config/prod/elevatems_features.yaml (hunk 2 above)
Comment (itismeghasyam) [reply config/prod/elevatems_features.yaml]: This is not a personal AMI. I just linked the tag for the latest Ubuntu AMI (US West Oregon) from http://www.louisaslett.com/RStudio_AMI/

I am unsure of the specifications i.e, root volume encryption

Comment (zaro0508) [reply config/prod/elevatems_features.yaml]: `us-east-1` would be better, which is  `ami-0226a8af83fcecb43`

**Pull request merged**